gradle/libs.versions.toml

```diff
@@ -24,7 +24,7 @@ androidxSplash = "1.1.0-rc01"
 androidXAppCompat = "1.7.0"
 androdixAutofill = "1.1.0"
 androidxWork = "2.9.1"
-bitwardenSdk = "1.0.0-20240924.112512-21"
+bitwardenSdk = "1.0.+"
```

Comment:
🗒️ [Note to self] Identify the exact version with SSH keys but not the manage collections version.